snippets/product-variant-picker.liquid

```diff
@@ -84,5 +84,7 @@
         </div>
       {%- endif -%}
     {%- endfor -%}
+
+    <script type="application/json" data-selected-variant>{{ product.selected_or_first_available_variant | json }}</script>
```

Comment:
This is a convenience to enable us to easily grab the JSON for the variant that is selected.